_posts/blog/2019-08-01-lab-meeting.md

```diff
@@ -0,0 +1,58 @@
+# Whitaker Lab Meeting | 1 August 2019
+
+This week’s lab meet was petite but by no means picayune! 
+We discussed ways to learn physics, Neurohackademy, and how to build trust and confidence in a group. 
+Thanks Kirstie, Isla, and Sarah for your tips, suggestions, and a super interesting conversation!
+
+### Celebrations and cool things to share:
+
+Kirstie: is finding Neurohackademy great so far – with lots of awesome talks! Here are a select few :point_down:
+  * Codeclimate for readable code: https://codeclimate.com
```

Comment:
I _think_ these are going to need to be formatted as links (GitHub does it automatically but jekyll won't know to convert them to links.

I've suggested for this first one and then you can do the rest.

```suggestion
  * Codeclimate for readable code: [https://codeclimate.com](https://codeclimate.com)
```